compiler/x/codegen/OMRCodeGenerator.cpp

```diff
@@ -3500,3 +3503,62 @@ OMR::X86::CodeGenerator::getOutOfLineCodeSize()
 
    return totalSize;
    }
+
+void
+OMR::X86::CodeGenerator::moveOutOfLineInstructionsToWarm()
```

Comment:
I think adding "Code" to the end of this function name makes it more descriptive: `moveOutOfLineInstructionsToWarmCode`

Reply:
done